### PR TITLE
Address component should not duplicate the word 'level' from ISS

### PR DESCRIPTION
**Pull request by DanielHeath**
This isn't quite there yet - need to get a product decision about what order to show unit, level & building (as well as confirm whether the `flat_unit` field contains just a number, or a number + description.

### CODE REVIEW

File: test/features/10-functional/10-basic/40-service-details.feature [hunk 1]
```diff
@@ -54,8 +54,21 @@ Feature: Service details page
         Then I should not see "You are ineligible if"
 
     Scenario: The address is a link to google maps
-        When I visit /service/866464
-        Then I can get to google maps by clicking "33 Elizabeth Street Richmond VIC 3121"
+       Given A service with:
+        ----------------------------------------------
+        * Location{}
```
Comment (danni) [inline test/features/10-functional/10-basic/40-service-details.feature]: Can we use something other than `{}` like `:` ?

File: test/support/dictionary.js [hunk 2]
```diff
@@ -26,7 +26,37 @@ function linesConverter(str: string, done: callback): void {
     done(null, str.split("\n"));
 }
 
+// Converts a key from the format used in features to an object key
+function sanitizeKey(key: string): string {
```
Comment (danni) [inline test/support/dictionary.js:30]: Is this really `_.string.slugify`?

File: test/support/dictionary.js [hunk 3]
```diff
@@ -26,7 +26,37 @@ function linesConverter(str: string, done: callback): void {
...
+     * Key | Value
+     */
+    var downcaseKey = arr => [sanitizeKey(arr[0]), arr[1]];
+
+    return _.object(
+        lines.map(
+            line => line.split("|").map(
+                cell => cell.trim()
+            )
+        ).map(downcaseKey)
```
Comment (danni) [inline test/support/dictionary.js]: You can combine this into one map.

File: test/support/dictionary.js [hunk 4]
```diff
@@ -69,14 +94,12 @@ function serviceConverter(str: string, done: callback): void {
     var serviceProps = {};
     var currentTableLines, // eg "Emails"
         currentTable,      // the lines which constitute a table
+        parseFn,// how should currentTableLines be parsed
```
Comment (danni) [inline test/support/dictionary.js]: Weird indenting.

File: test/features/10-functional/10-basic/40-service-details.feature (hunk 1 above)
Comment (DanielHeath) [inline test/features/10-functional/10-basic/40-service-details.feature]: We need a way to do objects, arrays and values - I've used the following for now:
`Key: Value` for bare values
`Key: (nada)` for empty values
`Key:` for arrays
`Key{}` for objects

Comment (DanielHeath) [inline test/features/10-functional/10-basic/40-service-details.feature]: Alternatively we could switch on the presence of header.

File: test/support/dictionary.js (hunk 2 above)
Comment (DanielHeath) [inline test/support/dictionary.js:30]: Doesn't `slugify` use dashes instead of underscores?

**Comment by DanielHeath** [comment]
@danni ++?

**Comment by danni** [comment]
++ :+1:

**Pull request merged**